examples/django_chat/chat/sockets.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from sdjango import namespace
+
+
+online_user_num = 0
+
+
+@namespace('/test')
+class TestNamespace:
+
+    def __init__(self, name):
+        self.name = name
+        self.request = None  # django request object
+
+    def _get_socket(self, sid):
+        socket = namespace.server.eio._get_socket(sid)
```

Comment:
yes, I undertstand `_` prefix's meaning in python. I think every client has its own socket so I want the django request instance to bind to it